climada/util/api_client.py

```diff
@@ -218,7 +227,7 @@ def _filter_datasets(datasets, multi_props):
             pdf = pdf[pdf[prop].isin(selection)]
         return [datasets[i] for i in pdf.index]
 
-    def get_datasets(self, data_type=None, name=None, version=None, properties=None,
+    def list_dataset_infos(self, data_type=None, name=None, version=None, properties=None,
```

Comment:
What does `name` refer too here? Is there a dataset property with is called `name` ?

Reply:
Is it the name of the file? or the uuid? or how is this name chosen/set?

Reply:
Arbitrary by the person who creates the dataset.

Reply:
I.g. the filename is most probably the dataset name plus file extension.